packages/nettacker/PKGBUILD

```diff
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=nettacker
+pkgver=0.0.2.r0.g92239ce
+pkgrel=1
+pkgdesc='Automate information gathering, vulnerability scanning and eventually generating a report for networks, including services, bugs, vulnerabilities, misconfigurations, and other information.'
+arch=('any')
+groups=('blackarch' 'blackarch-scanners')
+url='https://github.com/OWASP/Nettacker'
+license=('Apache')
+depends=('flake8' 'python-backoff' 'python-argparse' 'python-netaddr' 'python-ipaddr' 'python-requests' 'python-paramiko' 'python-texttable' 'python-pysocks' 'python-win_inet_pton' 'python-pyopenssl' 'python-flask' 'python-lockfile' 'python-bs4' 'python-pyspf' 'python-sqlalchemy' 'canari' 'python-py3dns' 'python-censys' 'python-six' 'python-shodan' 'python-pycurl' 'python-xmltodict' 'python-rule' 'python-mechanize' 'scapy' 'python-psycopg2')
+makedepends=('python-setuptools' 'git')
```

Comment:
Break long lines latest at 80 characters here and format the lines nicely. 

For reference you can have a look at `packages/python-taser/PKGBUILD`